extensions/interactions/NumericInput/directives/numeric_input_interaction_directive.html

```diff
@@ -10,6 +10,7 @@
       <!-- This ng-click is needed for specifying solutions in the solution editor -->
       <md-button ng-if="!isSubmitHidden()" class="oppia-learner-confirm-button"
                  ng-click="submitAnswer(answer)" style="margin-top: 15px;"
+                 ng-disabled="isAnswerValid()"
```

Comment:
Missing `!`?

Reply:
oops. sorry. I had tested it earlier and then changed the function from notValid to valid. forgot to negate it. Thanks

Reply:
Done. Thanks